### PR TITLE
More detailed guide to confining/wall potentials

### PR DESCRIPTION
**Pull request by awvwgk**
Currently the description on the usage of the wall potentials is incomplete.

We need to:

- [x]  rewrite the introduction to confining and wall potentials
- [x]  add information on placement of the potential aufpunkt
- [x]  add information on available potential shapes
- [x]  rewrite the part about anisotropic potentials
- [x]  add a description on usage of multiple potentials

@fabothch might want to look into this, too.

- [x] ready for merge

### CODE REVIEW

File: source/xcontrol.rst [hunk 1]
```diff
@@ -345,72 +345,154 @@ The detailed input looks like
...
   $wall
      potential=logfermi
      sphere: auto, all
   $end
 
-You can be more precise on the radius by giving the value in bohr instead
-of ``auto``. I personally recommend to use the logfermi potential, since it
-is best suited for confinements, but yet not the default.
+You can be more precise on the radius by giving the value in Bohr instead
+of ``auto``. The automatically determined radius is based on the largest
```
Comment (fabothch) [inline source/xcontrol.rst]: The largest distance from the cma along the axis?

Comment (awvwgk) [reply source/xcontrol.rst]: The largest interatomic distance is currently used.

**Comment by awvwgk** [comment]
there is the rest, let's try again, reopen #9

File: source/xcontrol.rst [hunk 2]
```diff
@@ -345,72 +345,154 @@ The detailed input looks like
...
-          implementation.
-          To avoid problems with misplaced confining potentials the ``$cma``
-          logical instruction should be included to shift the molecule back
-          to the center of mass and align it to its principal axes of inertia.
+.. code-block:: text
 
-**Example for using wall potentials:**
+   $wall
+      potential=logfermi
+      ellipsoid: 13.5,11.1,8.6,all # values in Bohr
```
[the remark this answers is not in the document]
Comment (awvwgk) [reply source/xcontrol.rst]: Comments are always removed from input files by xtb.

**Pull request merged**